patches/chrome-browser-ui-browser.cc.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
-index 939a669e01d34f83247786cd0a940cbc93cdc6f1..0a66f41df4fd551f907b4b4cadc4e7dda9dbe7e6 100644
---- a/chrome/browser/ui/browser.cc
-+++ b/chrome/browser/ui/browser.cc
-@@ -1121,7 +1121,7 @@ void Browser::SetFocusToLocationBar() {
-   //     the focus.  FocusLocationBar() is only reached when the location bar is
-   //     focusable, but this may be reached at other times, e.g. while in
-   //     fullscreen mode, where we need to leave focus in a consistent state.
--  window_->SetFocusToLocationBar();
-+  window_->SetFocusToLocationBar(false);
```

Comment:
I don't think this should be removed.

Reply:
All of these have been fixed in this bump (or rather reverted):
<pre>
commit 39d966addd91e03d7c203e1d4ca864997da74ab8
Author: Tommy C. Li <tommycli@chromium.org>
Date:   Wed Apr 10 19:49:59 2019 +0000

    Revert "Omnibox: Code health - Remove select_all parameter from FocusLocationBar"

    This reverts commit 8bb5bf47fcd1ca8158ca67540c2573b507cfcc8a.

    Reason for revert: We indeed need a select_all parameter for renderer
    initiated focuses. This is because if the user is faster than the NTP,
    and starts typing into the Omnibox before the NTP loads, it's bad for
    the NTP to trigger a select-all after it loads, as it clobbers user
    input.

    We can't just not select-all on the unfocused=>focused transition
    either, as users are accustomed to being able to press Ctrl+L to
    select everything in the omnibox even if it's already focused.

    Original change's description:
    > Omnibox: Code health - Remove select_all parameter from FocusLocationBar
    >
    > Currently there is a a select_all parameter that we plumb through all
    > layers of FocusLocationBar or its equivalent.
    >
    > This CL removes that parameter, and assumes it's always true. In other
    > words, any time we focus the location bar, we also select-all.
    >
    > This parameter is pretty much always set to true. The only place where
    > that parameter is set to false is from renderer-initiated focuses.
    >
    > Renderer-initiated focuses are used these scenarios:
    >  1. When the New Tab Page is loaded, in which case the omnibox is empty,
    >     and selecting all does nothing.
    >  2. For media OffscreenTab, there is no location bar, but the absent
    >     location bar is "focused" to prevent the page content from getting
    >     input focus. See OffscreenTab::ShouldFocusLocationBarByDefault.
    >  3. For the PresentationReceiverWindowController, there is also no
    >     location bar. It's also to prevent the page content from getting
    >     focus. See PresentationReceiverWindowController::
    >     ShouldFocusLocationBarByDefault.
    >  4. When a new tab loads with about:blank, this will change the behavior
    >     to automatically select-all the "about:blank" text when the
    >     location bar is auto-focused. This is a desirable change.
    >
    > In all these cases, setting select_all to true would have no negative
    > effect.
    >
    > Bug: 921777, 912722
    > Change-Id: Idefb493b196f89e23928e925ec9fcf015cac02fa
    > Reviewed-on: https://chromium-review.googlesource.com/c/1418450
    > Commit-Queue: Tommy Li <tommycli@chromium.org>
    > Reviewed-by: Peter Kasting <pkasting@chromium.org>
    > Reviewed-by: Alex Moshchuk <alexmos@chromium.org>
    > Cr-Commit-Position: refs/heads/master@{#624213}

    TBR=pkasting@chromium.org,tommycli@chromium.org,alexmos@chromium.org

    # Not skipping CQ checks because original CL landed > 1 day ago.

    Bug: 921777, 912722
    Change-Id: If693672c8ad7bb7a332f78d9695cb7a39ca83bd7
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1559115
    Reviewed-by: Tommy Li <tommycli@chromium.org>
    Reviewed-by: Peter Kasting <pkasting@chromium.org>
    Commit-Queue: Peter Kasting <pkasting@chromium.org>
    Cr-Original-Commit-Position: refs/heads/master@{#648955}(cherry picked from commit 43ef038d92433b1e8230882d3daac767481d29d9)
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1562211
    Cr-Commit-Position: refs/branch-heads/3729@{#775}
    Cr-Branched-From: d4a8972e30b604f090aeda5dfff68386ae656267-refs/heads/master@{#638880}

chrome/browser/autocomplete/autocomplete_browsertest.cc
chrome/browser/renderer_host/site_per_process_text_input_browsertest.cc
chrome/browser/ui/browser.cc
chrome/browser/ui/browser_commands.cc
chrome/browser/ui/browser_window.h
chrome/browser/ui/location_bar/location_bar.h
chrome/browser/ui/omnibox/omnibox_view_browsertest.cc
chrome/browser/ui/search/instant_extended_interactive_uitest.cc
chrome/browser/ui/views/frame/browser_view.cc
chrome/browser/ui/views/frame/browser_view.h
chrome/browser/ui/views/keyboard_access_browsertest.cc
chrome/browser/ui/views/location_bar/location_bar_view.cc
chrome/browser/ui/views/location_bar/location_bar_view.h
chrome/browser/ui/views/toolbar/toolbar_view.cc
chrome/test/base/test_browser_window.h
ui/aura/mus/window_tree_client_unittest.cc
</pre>